tools/update.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
```

Comment:
I don't want to check in code specialized for another product into mbed OS. Could this script be placed into your own repo?